serverlessworkflow/modules/ROOT/pages/eventing/handling-events-on-workflows.adoc

```diff
@@ -1 +1,147 @@
-//= Handling events on workflows
\ No newline at end of file
+= Handling events on workflows
+
+:compat-mode!:
+// Metadata:
+:description: Handling events on workflows
+:keywords: kogito, workflow, serverless, event
+
+This document describes Serverless Workflow link:{spec_doc_url}#Event-State[Event state]. Event state can be used to start a new workflow instance upon the reception of a designated link:{cloud_events_url}[CloudEvent] or to pause the execution of an existing workflow instance till a designated CloudEvent is received. 
+
+== Event State match 
+
+An Event state consist of one or more link:{spec_doc_url}#event-definition[event definitions]. Event definitions are combined to designate which CloudEvent Types the Event State should listen to. 
+
+`onEvents` field contains an array of `eventRefs`, which is an array of event definition names, and a set of actions to be executed when there is a match. `exclusive` field is used to determine which CloudEvent types should be considered to determine if there is an event match. If `exclusive` value is false, all CloudEvent types should be received for a match to occur. Otherwise, the reception of any of the CloudEvent Types is consider a match. 
```

Comment:
any suggestion? Because I do not think copying the spec text here is an option. Any summary of the spec (as I was trying) will be necessarily incomplete (and the spec itself has considerable gaps) . And the guidelines we are following considerely reduce my expression capabilities ;)

Reply:
I rewrote these parragraphs, but to be honest I prefer Heena to change them before changing them again.

Reply:
Looks good. I like the current version better, not sure though if it isn't because I've read it so many times.